src/xcode/ENA/ENA/Source/Services/HealthCertificate/Models/HealthCertificate.swift

```diff
@@ -90,9 +90,12 @@ final class HealthCertificate: Codable, Equatable, Comparable, RecycleBinIdentif
 
 	static func < (lhs: HealthCertificate, rhs: HealthCertificate) -> Bool {
 		if let lhsDate = lhs.sortDate, let rhsDate = rhs.sortDate {
-			return lhsDate < rhsDate
+			if lhsDate == rhsDate {
+				return lhs.cborWebTokenHeader.issuedAt < rhs.cborWebTokenHeader.issuedAt
+			} else {
+				return lhsDate < rhsDate
+			}
```

Comment:
```suggestion
			return (lhsDate, lhs.cborWebTokenHeader.issuedAt) < (rhsDate, rhs.cborWebTokenHeader.issuedAt)
```

This should work and looks a bit more elegant. Sorry, had a typo in it in my earlier comment (`lhs` in the right hand tuple), that's why it probably didn't work if you tried it out already 😅

Reply:
just tried it, it woks, nice!